charts/mageai/templates/secrets.yaml

```diff
@@ -0,0 +1,11 @@
+{{ if not .Values.existingSecret }}
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: {{ include "mageai.fullname" . }}-secert-env
```

Comment:
why not use "mageai.secretName" here?

Reply:
because it needed a good review. many thanks for spotting it!